transports/janus_rabbitmq.c

```diff
@@ -974,6 +1026,12 @@ void *janus_rmq_out_thread(void *data) {
 	}
 	JANUS_LOG(LOG_VERB, "Joining RabbitMQ out thread\n");
 	while(!rmq_client->destroy && !g_atomic_int_get(&stopping)) {
+
+		while (!rmq_client->connected) {
+			JANUS_LOG(LOG_WARN, "Waiting for 5s for RabbitMQ to reconnect\n");
+			g_usleep(5000000);
```

Comment:
I'm not sure about this big sleep here... wouldn't it make sense to have some sort of exponential backoff, that starts with a much shorter interval instead?